src/Aks/Aks/ChangeLog.md

```diff
@@ -26,6 +26,8 @@
 * Added parameter `-NodePoolTag` for `New-AzAksCluster` and `Set-AzAksCluster`, `-Tag` for `New-AzAksNodePool` and `Update-AzAksNodePool`
 * Added parameter `-NodeTaint` for `New-AzAksNodePool` and `Update-AzAksNodePool`
 * Added parameter `-AksCustomHeader` for `Set-AzAksCluster`, `New-AzAksNodePool` and `Update-AzAksNodePool`
+* Added parameter `-EnableEncryptionAtHost` for `New-AzAksCluster` and `New-AzAksNodePool`
```

Comment:
Please move change logs between L20-21 so that we can correctly place them in the upcoming release section after release-2022-12-06 merge back.